### PR TITLE
added the dropdown list data

### PR DESCRIPTION
**Pull request by jameka23**
added the intermediary tables via data arrays

### CODE REVIEW

File: data/moods.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const moods = [
+    { value: 'excited', label: 'Excited'},
+    { value: 'happy', label: 'Happy'},
+    { value: 'meh', label: 'Meh'},
+    { value: 'sad', label: 'Sad'},
+    { value: 'Mad', label: 'Mad'},
```
Comment (drteresavasquez) [inline data/moods.js]: @jameka23 maybe change value to lowercased for consistency?

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
One small change, otherwise looks great!

File: data/moods.js (hunk 1 above)
Comment (jameka23) [reply data/moods.js]: @drteresavasquez ahh good check

**Review by drteresavasquez — APPROVED** [review]
👏🏼👏🏼

**Pull request merged**